### PR TITLE
add mercury dashboarding tool

### PR DESCRIPTION
**Pull request by pplonski**
Hi!

I would like to add [Mercury](https://github.com/mljar/mercury) framework as a dashboarding tool. It can convert Jupyter Notebook to a web app (dashboard/report/slides) with interactive widgets. You don't need to code the UI. It is generated from the YAML header and directly connected to variables in Python code.

Thank you!
Piotr

### CODE REVIEW

File: doc/dashboarding/index.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 
 - [Dash](https://plot.ly/products/dash) (from [Plotly](https://plot.ly)); see the [blog post](https://medium.com/@plotlygraphs/introducing-dash-5ecf7191b503)
 - [Panel](https://panel.pyviz.org) (from [Anaconda](http://anaconda.com)); see the [blog post](https://medium.com/@philipp.jfr/panel-announcement-2107c2b15f52)
 - [Voila](https://github.com/QuantStack/voila) (from [QuantStack](http://quantstack.net)); see the [blog post](https://blog.jupyter.org/and-voil%C3%A0-f6a2c08a4a93); used with separate layout tools like 
 [jupyter-flex](https://github.com/danielfrg/jupyter-flex) or templates like [voila-vuetify](https://github.com/voila-dashboards/voila-vuetify).
 - [Streamlit](https://www.streamlit.io); see the [blog post](https://towardsdatascience.com/coding-ml-tools-like-you-code-ml-models-ddba3357eace)
+- [Mercury](https://github.com/mljar/mercury) (from [MLJAR](https://mljar.com)); see the [blog post](https://mljar.com/blog/dashboard-python-jupyter-notebook/)
```
Comment (jbednar) [inline doc/dashboarding/index.md:10]:
```suggestion
```

Comment (jbednar) [inline doc/dashboarding/index.md:10]:
```suggestion
```

File: doc/dashboarding/index.rst [hunk 2]
```diff
@@ -8,3 +8,4 @@
    Dash <https://plot.ly/products/dash>
    Panel <https://panel.pyviz.org>
    Voila <https://github.com/QuantStack/voila>
+   Mercury <https://github.com/mljar/mercury>
```
Comment (jbednar) [inline doc/dashboarding/index.rst:11]:
```suggestion
```

**Review by jbednar — CHANGES_REQUESTED** [review]
Thanks for your contribution to PyViz! We should definitely list mercury in the all-tools list, and this PR looks good for doing that.

However, I've made suggestions that remove the changes to the dashboarding page that list mercury as one of the "five main tools", because I can't see any argument that would place mercury ahead of the other eight tools also not included as "main" tools. Other than gradio, those other tools all have 60K or fewer downloads per month, with mercury far down in that list at 2K per month, compared to 500K per month for those in the main list. 

In any case, certainly good to have the list be more complete, and happy to merge it once those suggestions are applied!

File: doc/dashboarding/index.md [hunk 3]
```diff
@@ -1,12 +1,13 @@
 # Dashboarding tools
 
-Just about any Python library can be used to create a "static" PNG, SVG, HTML, or other output that can be pasted into a presentation, sent in an email, published as a figure in a paper, and so on.  Many people also want or need to create "live" Python-backed applications or dashboards that a user can interact with to explore or analyze some data. Python offers several libraries for this purpose. The four main tools designed specifically for web-based dashboarding in Python are:
+Just about any Python library can be used to create a "static" PNG, SVG, HTML, or other output that can be pasted into a presentation, sent in an email, published as a figure in a paper, and so on.  Many people also want or need to create "live" Python-backed applications or dashboards that a user can interact with to explore or analyze some data. Python offers several libraries for this purpose. The five main tools designed specifically for web-based dashboarding in Python are:
```
Comment (jbednar) [inline doc/dashboarding/index.md:3]:
```suggestion
Just about any Python library can be used to create a "static" PNG, SVG, HTML, or other output that can be pasted into a presentation, sent in an email, published as a figure in a paper, and so on.  Many people also want or need to create "live" Python-backed applications or dashboards that a user can interact with to explore or analyze some data. Python offers several libraries for this purpose. The four main tools designed specifically for web-based dashboarding in Python are:
```

**Comment by pplonski** [comment]
Thanks @jbednar! I applied your suggestions. You are right, Mercury is not yet as popular as other frameworks. Mercury is only 6 months old. It is the only solution to convert Jupyter Notebook to the interactive dashboard without additional coding (Voila requires coding with ipywidgets). I hope it will join the big four in the future :)

**Pull request merged**